.vscode/launch.json

```diff
@@ -6,27 +6,27 @@
   "configurations": [
     {
       "command": "npm start",
-      "name": "Launch web app with CM API only",
+      "name": "Launch web app with Intersection Data",
       "request": "launch",
       "cwd": "${workspaceFolder}/webapp",
       "type": "node-terminal",
-      "preLaunchTask": "run-conflictmonitor-api-only"
+      "preLaunchTask": "run-intersection"
     },
     {
       "command": "npm start",
-      "name": "Launch web app with CM",
+      "name": "Launch web app without Intersection Data",
       "request": "launch",
       "cwd": "${workspaceFolder}/webapp",
       "type": "node-terminal",
-      "preLaunchTask": "run-conflictmonitor"
+      "preLaunchTask": "run-keycloak-and-postgres"
     },
     {
       "command": "npm start",
-      "name": "Launch web app without CM",
+      "name": "Launch web app with Full ConflictMonitor",
       "request": "launch",
       "cwd": "${workspaceFolder}/webapp",
       "type": "node-terminal",
-      "preLaunchTask": "run-keycloak-and-postgres"
+      "preLaunchTask": "run-full-conflictmonitor"

```

Comment:
It looks like this task call out launches the full `docker-compose-full-cm.yml` which includes a web app instance, are we duplicating the web app here?

Reply:
Fixed